src/View/View.php

```diff
@@ -1244,7 +1244,27 @@ public function helpers(): HelperRegistry
     }
 
     /**
-     * Loads a helper. Delegates to the `HelperRegistry::load()` to load the helper
+     * Adds a helper from within `initialize()` method.
+     *
+     * @param string $helper Helper.
+     * @param array<string, mixed> $config Config.
+     * @return void
+     */
+    protected function addHelper(string $helper, array $config = []): void
+    {
+        [$plugin, $name] = pluginSplit($helper);
+        if ($plugin) {
+            $config['class'] = $helper;
+            $config['config'] = $config;
```

Comment:
```suggestion
            $config['className'] = $helper;
```
This form is better as that's what's currently used for `$helpers` property.

Reply:
I am dropping the `['class' => 'Foo', 'config' => []]` form in 5.x altogether since `ObjectRegistry` support `['className' => 'Foo', .. other config ..]` form just fine.